inventory

```diff
@@ -0,0 +1,3 @@
+[target] 
```

Comment:
I'd be more specific here. Also it's a group so maybe plural is appropriate.
[dev-machines]